packages/office-ui-fabric-react/src/components/Keytip/KeytipPage.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
```

Comment:
The documentation for Keytips will be consolidated to one page. In general the documentation is very incomplete, will be updated of course before this is all completed. I'm currently focused on integration with the ribbon which is helping verify functionality